### ISSUE TITLE
Organization column label spacing off on Users->Teams list

### ISSUE THREAD

**Issue by mabashian** [ansible/awx#7563]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
I believe the spacing between the label (Organization) and the value should look like:

<img width="1346" alt="Screen Shot 2020-07-07 at 9 44 10 AM" src="https://user-images.githubusercontent.com/9889020/86790433-6eecc880-c036-11ea-9c97-4948804b3043.png">

instead it looks like:

<img width="1680" alt="Screen Shot 2020-07-07 at 9 42 31 AM" src="https://user-images.githubusercontent.com/9889020/86790455-74e2a980-c036-11ea-9d68-11ea92e6d18b.png">

tldr; there should be 24px between `Organization` and `Default` in the Users->Teams list to match other label/value pairs in lists.

**Comment by mabashian** [comment]
Same thing applies to the regular Teams list (probably bc it's the same component)

**Comment by mabashian** [comment]
Marking as blocked pending #6189

**Comment by tiagodread** [comment]
@keithjgrant I realised that this list `User -> Organization Tab` is not using the new tables list component like the `Organizations` list is. I don't know if we have a specific issue to keep track on all this remaining lists or not.

**Comment by keithjgrant** [comment]
> @keithjgrant I realised that this list `User -> Organization Tab` is not using the new tables list component like the `Organizations` list is. I don't know if we have a specific issue to keep track on all this remaining lists or not.

We're tracking that under #6189

**Comment by mabashian** [comment]
With the new list implementation I think we can close this one out
